packages/hash/integration/src/tests/model/knowledge/link.model.test.ts

```diff
@@ -0,0 +1,148 @@
+import { getRequiredEnv } from "@hashintel/hash-backend-utils/environment";
+import { createGraphClient } from "@hashintel/hash-api/src/graph";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+
+import {
+  EntityModel,
+  EntityTypeModel,
+  LinkModel,
+  LinkTypeModel,
+} from "@hashintel/hash-api/src/model";
+
+jest.setTimeout(60000);
+
+const logger = new Logger({
+  mode: "dev",
+  level: "debug",
+  serviceName: "integration-tests",
+});
+
+const graphApiHost = getRequiredEnv("HASH_GRAPH_API_HOST");
+const graphApiPort = parseInt(getRequiredEnv("HASH_GRAPH_API_PORT"), 10);
+
+const graphApi = createGraphClient(logger, {
+  host: graphApiHost,
+  port: graphApiPort,
+});
+
+const accountId = "00000000-0000-0000-0000-000000000000";
+
+const entityType$id = "https://link~example.com/entity-type/v/1";
+
+describe("Link model class", () => {
+  let testType: EntityTypeModel;
+  let linkTypeFriend: LinkTypeModel;
+  let linkTypeAcquaintance: LinkTypeModel;
+  let sourceEntity: EntityModel;
+  let targetEntityFriend: EntityModel;
+  let targetEntityAcquaintance: EntityModel;
+
+  beforeAll(async () => {
+    testType = await EntityTypeModel.create(graphApi, {
+      accountId,
+      schema: {
+        $id: entityType$id,
+        kind: "entityType",
+        title: "Text",
```

Comment:
I know this is just a test and we've been pretty poor about adding meaningful types but I think calling an _entity_ type `Text is even more confusing than is necessary at this point 😄 can we call it something else?

Reply:
Addressed in c3791a565d1b70a5f1813a42cc3bf1085ff25a47